internal/db/db_blueprints.go

```diff
@@ -25,6 +25,19 @@ const (
 		ORDER BY blueprint_versions.created_at DESC LIMIT 1`
 
 	sqlDeleteBlueprint = `DELETE FROM blueprints WHERE id = $1 AND org_id = $2 AND account_number = $3`
+
+	sqlGetBlueprints = `
+		SELECT blueprints.id, blueprints.name, blueprints.description, MAX(blueprint_versions.version) as version, MAX(blueprint_versions.created_at) as last_modified_at
+		FROM blueprints INNER JOIN blueprint_versions ON blueprint_versions.blueprint_id = blueprints.id
+		WHERE blueprints.org_id = $1 AND blueprints.account_number = $2
+		GROUP BY blueprints.id
+		ORDER BY last_modified_at DESC
```

Comment:
This needs to be:

```suggestion
		ORDER BY blueprint_versions.created_at DESC
```

your code enforces the SQL engine to sort after grouping and since the calculated column only contains maximum timestamp, it would actually not work if I read the query correctly.

Reply:
I'm getting
```
 column "blueprint_versions.created_at" must appear in the GROUP BY clause or be used in an aggregate function (SQLSTATE 42803)
```
But ordering by the `last_modified_at` should work, it aggregate the maximum timestamp for each blueprints_version, but the order relation exists across distinct blueprints.